osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/entities/DeviceModel.java

```diff
@@ -37,28 +37,22 @@
     @Column
     private boolean fileStorage;
 
-    @Column
-    private boolean metered;
-
     public DeviceModel() {
         // Default constructor
     }
 
-    public DeviceModel(final Manufacturer manufacturer, final String modelCode, final String description,
-            final boolean metered) {
+    public DeviceModel(final Manufacturer manufacturer, final String modelCode, final String description) {
         this.manufacturer = manufacturer;
         this.modelCode = modelCode;
         this.description = description;
         // default behavior is true
         this.fileStorage = true;
```

Comment:
```suggestion
        this(manufacturer, modelCode, description, true);
```

Reply:
It is not possible to have constructor a call constructor b, while constructor b calls constructor a. Not changing this code.

Reply:
Updated.